sdk/test/opentelemetry/sdk/trace/span_test.rb

```diff
@@ -200,14 +201,41 @@
       _(events.first.attributes).must_equal(attrs)
     end
 
-    it 'add event with timestamp' do
+    it 'uses the provided timestamp' do
       ts = Time.now
```

Comment:
Can we use something completely weird here to ensure we're not getting the realtime or monotonic timestamps instead?

Reply:
✅ Explicit event timestamp is honoured